lab_2_keywords_cooccurrence/main.py

```diff
@@ -123,8 +225,34 @@ def extract_candidate_keyword_phrases_with_adjoining(candidate_keyword_phrases:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
-
+    if not (candidate_keyword_phrases and phrases):
+        return None
+    #тут будет проверка на тип
+    possible_pairs = {}
+    for i in range(len(candidate_keyword_phrases) - 1):
+        pair = candidate_keyword_phrases[i], candidate_keyword_phrases[i + 1]
+        if pair not in possible_pairs:
+            possible_pairs[pair] = 0
```

Comment:
i don't yet get why you initialize value as 0 
could you explain the logic

Reply:
i'm sorry, i made a mistake. it's supposed to be 1